src/_data/schemas/lpa.json

```diff
@@ -0,0 +1,289 @@
+                    "$ref": "#/definitions/Witness",
+                    "description": "The Witness for the LPA.",
+                    "title": "Witness on the LPA"
+                },
+                "certificateProvider": {
+                    "$ref": "#/definitions/CertificateProvider",
+                    "description": "The Certificate Provider on the LPA.",
+                    "title": "Certificate Provider on the LPA"
+                },
+                "attorney": {
+                    "type": "array",
+                    "title": "All attorney(s) on the LPA",
+                    "description": "The Attorney(s) on the LPA.",
+                    "items": {
+                        "$ref": "#/definitions/Attorney"
+                    }
+                },
+                "personToNotify": {
+                    "type": "array",
+                    "title": "Person(s) to Notify",
+                    "description": "The Person(s) to Notify on the LPA.",
+                    "items": {
+                        "$ref": "#/definitions/PersonToNotify"
+                    }
+                }
+            },
+            "required": [
+                "donor",
+                "lpaid",
+                "attorney"
+            ],
+            "title": "Lasting Power of Attorney"
+        },
+        "Donor": {
+            "type": "object",
+            "description": "The Donor.",
+            "additionalProperties": false,
+            "properties": {
+                "actor": {
+                    "$ref": "#/definitions/Actor"
+                },
+                "signature": {
+                    "$ref": "#/definitions/Signature"
+                }
+            },
+            "required": [
+                "actor"
+            ],
+            "title": "Donor"
+        },
+        "Attorney": {
+            "type": "object",
+            "description": "The Attorney.",
+            "additionalProperties": false,
+            "properties": {
+                "actor": {
+                    "$ref": "#/definitions/Actor"
+                },
+                "signature": {
+                    "$ref": "#/definitions/Signature"
+                }
+            },
+            "required": [
+                "actor"
+            ],
+            "title": "Attorney"
+        },
+        "Witness": {
+            "type": "object",
+            "description": "The Witness to the signing of the LPA.",
+            "additionalProperties": false,
+            "properties": {
+                "actor": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/Actor"
+                    }
+                },
+                "signature": {
+                    "$ref": "#/definitions/Signature"
+                }
+            },
+            "required": [
+                "actor"
+            ],
+            "title": "Witness"
+        },
+        "CertificateProvider": {
```

Comment:
Certificate providers can be trust corporations at the moment, rather than actors. We need to be able to provide for something like this I'd imagine.